### DISCUSSION TITLE
Potential opportunity to reduce file sizes for billing class identifiers

### DISCUSSION THREAD

**Discussion by brbucher** [CMSgov/price-transparency-guide#366] [Ideas]
Based on our analysis, the code types of APC, DRG, and RC would all be institutional only.
All of the other code types would appear for both professional and institutional and produce the same rate in our system.
Considering this, could we list "Professional" and "Institutional" in the same object to avoid producing duplicate objects for codes where the prices would be the same for both?

**Comment by jleberre10** [comment]
Agree that the option to list both would make sense and help with file size. Good suggestion.

**Reply by rakeshkatte** [reply]
@shaselton-usds  This is a good suggestion, can the "both" value be added. Even we have similar situation where we will have to publish duplicate info due to the billing_class.

**Comment by brbucher** [comment]
Bumping this thread, any thoughts @shaselton?

**Comment by shaselton-usds** [comment]
@brbucher How would this impact the assigning of a `service_code` to a contract when it might not be applicable to both institutional and professional billing classes?

**Reply by brbucher** [reply]
As long as we have the three options of "professional", "institutional", or "both" that should provide the flexibility necessary to still properly represent scenarios where the service codes need to be distinct.
